Header header

bool storage_critical;
int64_t storage_total_bytes;
int64_t storage_available_bytes;
